src/Sulu/Bundle/MarkupBundle/Listener/MarkupListener.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MarkupBundle\Listener;
+
+use Sulu\Bundle\MarkupBundle\Markup\MarkupParserInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+
+/**
+ * Parses content of response and set the replaced html as new content.
+ */
+class MarkupListener
+{
+    /**
+     * @var MarkupParserInterface
+     */
+    private $markupParser;
+
+    /**
+     * @var array
+     */
+    private $mimeTypeFormatMap;
+
+    /**
+     * @param MarkupParserInterface[] $markupParser
+     * @param array $mimeTypeFormatMap
+     */
+    public function __construct(
+        array $markupParser,
+        array $mimeTypeFormatMap = ['text/html' => 'html', 'application/xml' => 'xml', 'text/xml' => 'xml']
```

Comment:
I would not pass this array here, because it is not extendable. For this reason I would place the defaults in the DI configuration.